deploying/deploying-to-heroku.md

```diff
@@ -115,50 +115,39 @@ if (process.env.NODE_ENV === 'production') {
 
 This block of code only runs in production. When it runs, it will serve your JavaScript files if the URL doesn't match an existing API.
 
-6. Configure the top-level `package.json` to work with Heroku by adding the following two lines to the `scripts` section:
+7. Configure the top-level `package.json` to work with Heroku. Add the following two lines to the json section:
 
-```json
-    "start": "node server/server.js",
+```
+    "start": "node server/<filename.js>",
```

Comment:
I opted for "filename" because in the student repo I used to follow the tutorial the server entry was at `index.js` not `app.js`. My intention was to eliminate potential confusion. I defer to your understanding of the project in its entirety for what is most effective for student learning.

Reply:
I opted for `<filename>.js` to indicate that the student's filename may be something else. However, if it will always be named `server.js` then my suggestion isn't useful.